shared-libs/cht-datasource/src/person.ts

```diff
@@ -59,4 +80,12 @@ export namespace v1 {
    * @throws Error if the provided context or qualifier is invalid
    */
   export const getWithLineage = getPerson(Local.Person.v1.getWithLineage, Remote.Person.v1.getWithLineage);
+
+  /**
+   * Returns an array of people.
+   * @param context the current data context
+   * @returns an array of people
+   * @throws Error if the provided context or personType qualifier is invalid
+   */
```

Comment:
As I mentioned in the comment above, after seeing how the docs site gets generated for this file, I am thinking we should change our documentation pattern a bit here.  This function documentation can specifically indicate it returns _a function_ and then we can get the documentation for the returned function _to be pulled through_ and also displayed on the site.

```suggestion
  /**
   * Returns a function for retrieving a paged array of persons from the given data context.
   * @param context the current data context
   * @returns a function for retrieving a paged array of persons
   * @throws Error if a data context is not provided
   */
```